sdks/java/core/src/main/java/org/apache/beam/sdk/coders/CoderFactory.java

```diff
@@ -18,27 +18,23 @@
 package org.apache.beam.sdk.coders;
 
 import java.util.List;
+import org.apache.beam.sdk.values.TypeDescriptor;
 
 /**
- * A {@link CoderFactory} creates coders and decomposes values.
- * It may operate on a parameterized type, such as {@link List},
- * in which case the {@link #create} method accepts a list of
- * coders to use for the type parameters.
+ * A {@link CoderFactory} creates coders.
+ *
+ * <p>It may operate on a parameterized type, such as {@link List}, in which case the
+ * {@link #create} method accepts a list of coders to use for the type parameters.
```

Comment:
I guess if users are going to implement `CoderFactory` we should convert to abstract class or provide a base class that we tell them to extend. It makes me sad, since interfaces are so much more flexible in other ways and are really the right tool, but... Java.